#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_X99_is_not_stable.dpatch by Paul Hampson <Paul.Hampson@anu.edu.au>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: rlm_x99 (X.99 token support) is not stable

@DPATCH@

--- freeradius-1.0.2.orig/src/modules/stable
+++ freeradius-1.0.2/src/modules/stable
@@ -25,5 +25,4 @@
 rlm_realm
 rlm_sql
 rlm_unix
-rlm_x99_token
 rlm_checkval
